.travis.yml

```diff
@@ -4,14 +4,11 @@ language: bash
 services: docker
 
 env:
-  - TERRAFORM_VERSION=0.9.6
-  - TERRAFORM_VERSION=0.9.8
-  - TERRAFORM_VERSION=0.9.11
```

Comment:
These images are likely still in use. They won't disappear from Quay, they will just cease being updated.